### PR TITLE
[best practices] mostly typos

### PR DESCRIPTION
**Pull request by talitakz**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | x |

### CODE REVIEW

**Comment by paxyknox** [comment]
I made a mistake!! I request merge into 2.3! I apologize.

**Comment by javiereguiluz** [comment]
@paxyknox no worries! Selecting the wrong branch is something that happen to all of us sometimes :disappointed:

Thanks for fixing all these typos and thanks for contributing a small pull request, which makes it much easier to review. Thanks!

File: best_practices/controllers.rst [hunk 1]
```diff
@@ -80,7 +80,7 @@ The ``@Template`` annotation is useful, but also involves some magic. We
 don't think its benefit is worth the magic, and so recommend against using
 it.
 
-Most of the time, ``@Template`` is used without any parameters, which makes
+Most of the time, ``@Template()`` is used without any parameters, which makes
```
Comment (WouterJ) [inline best_practices/controllers.rst]: As I explained in another great PR of you, I don't like using `()` for annotations.

File: best_practices/forms.rst [hunk 2]
```diff
@@ -160,8 +160,8 @@ thing in one line to rendering each part of each field independently. The
 best way depends on how much customization you need.
 
 One of the simplest ways - which is especially useful during development -
-is to render the form tags and use ``form_widget()`` to render all of the
-fields:
+is to render the form tags and use Twig ``form_widget()`` function to render
```
Comment (WouterJ) [inline best_practices/forms.rst]: I would use either "`form_widget()` Twig function" or just "`form_widget()` function". I like the second one most, as "Twig" doesn't add anything more to the sentence.

**Comment by paxyknox** [comment]
I made requested changes.
If there are no other remarks this could probably be merged.

**Comment by xabbuh** [comment]
Thank you @paxyknox. Very nice changes again! :+1: I have merged them into the `2.3` branch and made some small tweaks in 037c7a56b16c726ac6ef82663f1e4defd9b6f648.

**Pull request closed without merging**